server/config/options.go

```diff
@@ -67,6 +67,8 @@ type AuthConfig struct {
 	ManagementClientId        string        `mapstructure:"management_client_id" yaml:"management_client_id" json:"management_client_id"`
 	ManagementClientSecret    string        `mapstructure:"management_client_secret" yaml:"management_client_secret" json:"management_client_secret"`
 	TokenClockSkewDurationSec int           `mapstructure:"token_clock_skew_duration_sec" yaml:"token_clock_skew_duration_sec" json:"token_clock_skew_duration_sec"`
+	GotrueUrl                 string        `mapstructure:"gotrue_url" yaml:"gotrue_url" json:"gotrue_url"`
+	GotrueUsernameSuffix      string        `mapstructure:"gotrue_username_suffix" yaml:"gotrue_username_suffix" json:"gotrue_username_suffix"`
```

Comment:
Maybe these are from another change?

Reply:
ah right this is from other changes. Fixed it.